mitosheet/mitosheet/step.py

```diff
@@ -144,6 +145,8 @@ def set_prev_state_and_execute(self, new_prev_state: State) -> bool:
         execution actually occured.
         """        
         # Saturate the event to get up to date parameters
+        # TODO: this should fill in the execution data - hopefully
+        # we can get all of it without executing. I think we probably can
         params = self.step_performer.saturate(new_prev_state, self.params)
```

Comment:
Remove the saturate function? I think it's just execute now. I think we want to get rid of it!